app/src/main/java/com/movies/presentation/home/view_model/HomeViewModel.kt

```diff
@@ -1,7 +1,27 @@
 package com.movies.presentation.home.view_model
 
-import androidx.lifecycle.ViewModel
+import com.movies.common.extensions.viewModelScope
+import com.movies.common.utils.LiveDataDelegate
+import com.movies.domain.usecase.movies.GetMoviesUseCase
+import com.movies.presentation.base.view_model.BaseViewModel
+import com.movies.presentation.model.mapper.MovieUIMapper
+import com.movies.presentation.model.movie.MovieUIModel
 
-class HomeViewModel : ViewModel() {
+class HomeViewModel(
+    private val moviesUseCase: GetMoviesUseCase,
+    private val moviesUIMapper: MovieUIMapper
+) : BaseViewModel() {
+
+    val getMoviesLiveData by LiveDataDelegate<List<MovieUIModel>?>()
```

Comment:
Change name and remove nullable parameter. Use default values for nullable data

Reply:
done